samples/bookinfo/src/productpage/requirements.txt

```diff
@@ -3,7 +3,7 @@ chardet==3.0.4
 Click==7.0
 contextlib2==0.5.5
 dominate==2.3.5
-Flask==2.3.2
+Flask==2.1.1
```

Comment:
Is this an extra change that downgrades back to 2.1.1?

Reply:
Correct. This was necessary in order to build to proceed, at least in my machine. After this change gets merged, we can always bump versions (even automated ones like this) with confidence the build will not break.

Reply:
The Flask upgrade breaks the build on my machine as well. I did spend some time trying to upgrade other requirements to fix the errors, and so far have not been successful. If there is a Python developer who can upgrade the various requirements and still have things build, that would be great.

Reply:
+1. That could be done in a follow up as bookinfo is just sample, not supported and not meant to go in production. Low priority.

Reply:
Agree with doing it in a later PR.